frontend/ui/translator.lua

```diff
@@ -489,7 +489,20 @@ function Translator:_showTranslation(text, target_lang, source_lang)
                     table.insert(output, symbol .. " " .. t)
                 end
             end
-            table.insert(output, "")
+        end
+    end
+
+    if result[13] and type(result[13]) == "table" and #result[13] > 0 then
+        -- Definition(word)
+        table.insert(output, "________")
+        for i, r in ipairs(result[13]) do
+            if r[2] and type(r[2]) == "table" then
+                local symbol = util.unicodeCodepointToUtf8(10101 + (i < 10 and i or 10))
```

Comment:
More than 10 is unusual?

Reply:
There shouldn't be more than 10, as it puts number before parts of speech(noun/verb/etc.)
according to wikipedia there should be 9. 8 without articles. I copied this part from above and left this check just in case.
`local symbol = util.unicodeCodepointToUtf8(10101 + i)` *should* be alright too.

Reply:
How do you mean according to Wikipedia there should be 9? Nine of what?

Reply:
https://en.wikipedia.org/wiki/Part_of_speech
9 parts of speech when you look at the sceenshot in PR it's formatted like this
[symbol1]Noun
     meaning 1
     meaning 2
[symbol2]Verb
     meaning 3
     meaning

Reply:
Once you list them all: 
    noun
    verb
    adjective
    adverb
    pronoun
    preposition
    conjunction
    interjection
    article
you get 9 top, that is assuming one word can be all of the above.

Reply:
Oh, this code is copied from a few lines up and it's only about that. Your screenshot also shows synonyms or some such for example.